packages/js/product-editor/src/components/product-mvp-feedback-modal/style.scss

```diff
@@ -1,23 +1,62 @@
+$modal-header-height: 84px;
+
 .woocommerce-product-mvp-feedback-modal {
+	width: 600px;
+
+	.components-modal__header {
+		height: $modal-header-height;
+
+		&-heading {
+			font-weight: 500;
+			font-size: 20px;
+		}
+	}
+
+	.components-modal__content {
+		margin-top: $modal-header-height;
+		padding-bottom: 32px;
+	}
+
+	legend,
+	label {
+		color: $gray-900;
+	}
+
+	.woocommerce-product-mvp-feedback-modal__optional {
+		color: $gray-700;
+	}
+
+	legend {
+		font-size: 11px;
```

Comment:
I guess it's a question instead of a suggestion: do we have any standards for font-sizes? 11px seems odd (literally). I see it's the same size of the design.

Reply:
We've used it in a lot of places, but I'm not sure if there is a style guide somewhere for this.  It would be really nice to create variables around this (e.g., `$font-size-label`, `$font-size-help`, etc).  Not sure if something like this already exists on the design side.  cc @jarekmorawski